wasm/src/account/signature.rs

```diff
@@ -48,6 +48,18 @@ impl Signature {
         Self(SignatureNative::sign_bytes(private_key, message, &mut StdRng::from_entropy()).unwrap())
     }
 
+    /// Convert a message to field representation and sign the fields with a private key
+    ///
+    /// @param {PrivateKey} private_key The private key to sign the message with
+    /// @param {String} message The message to sign
+    /// @returns {Signature} Signature of the message
+    pub fn sign_fields(private_key: &PrivateKey, message: &str) -> Result<Self, String> {
```

Comment:
Please change this to sign_value per the reasoning above.

```suggestion
    pub fn sign_value(private_key: &PrivateKey, message: &str) -> Result<Self, String> {
```